.drone.yml

```
@@ -345,12 +345,12 @@ pipeline:
       - latest
       - ${DRONE_TAG##release-}
 
-  build-idpoller-release:
+  build-identities-worker-release:
     group: release2
     image: plugins/docker
-    dockerfile: src/backend/Dockerfile.identity-poller
+    dockerfile: src/backend/Dockerfile.identities-worker
     context: /srv/caliopen/src/backend
-    repo: registry.caliopen.org/caliopen_identity_poller
+    repo: registry.caliopen.org/caliopen_identities-worker
```

```suggestion
    repo: registry.caliopen.org/caliopen_identities_worker
```

Dockerfiles use dashes but Docker repositories use exclusively underscores.